src/scripts/main.js

```diff
@@ -1 +1,71 @@
 'use strict';
+
+const logo = document.querySelector('.logo');
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => reject(new Error('First promise was rejected')), 3000);
+});
+
+firstPromise
+  .then((message) => {
+    addMessage('success', message);
+  })
+  .catch((error) => {
+    addMessage('success', error);
```

Comment:
```suggestion
    addMessage('error', error);
```
Почему тут тоже success, если это ошибка?